src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo;
+
+import com.example.demo.model.Match;
+import com.example.demo.repository.TeamRepository;
+import com.example.demo.model.Team;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+
+import java.util.Scanner;
+
+@SpringBootApplication
+@ComponentScan
+public class DemoApplication implements CommandLineRunner {
+
+    public static void main(String[] args) {
+        SpringApplication.run(DemoApplication.class, args);
+    }
+
+    @Override
+    public void run(String... args) {
+        Menu();
+    }
+
+    public void Menu() {
```

Comment:
According java code conventions, method names should start with lower case letter and follow camel case (https://wiki.c2.com/?LowerCamelCase)